openshift_pipelines/installing-pipelines.adoc

```diff
@@ -0,0 +1,32 @@
+[id="installing-pipelines"]
+= Installing OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: installing-pipelines
+
+toc::[]
+
+// Prerequisites for installing OpenShift Operator
+[discrete]
+.Prerequisites
+
+* You have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+* You have installed xref:../cli_reference/openshift_cli/getting-started-cli.adoc[`oc` CLI].
```

Comment:
I haven't usually seen use an xref for `oc` CLI. I don't have strong feelings either way, although the blue link text on 'oc' looks off to me against the background color.

Reply:
I checked at other places in the docs, and have removed the the xref to make it consistent with others